examples/api_pipeline_run.py

```diff
@@ -22,8 +20,6 @@ def add_lol(a: str) -> str:
 
 run_result = api.run_pipeline(upload_output, "Hi I like to")
 print("Run state:", run_result["run_state"])
-try:
-    result_preview = json.loads(run_result["result_preview"])
-except (TypeError, json.JSONDecodeError):
-    result_preview = "unavailable"
+
+result_preview = run_result["result_preview"]
```

Comment:
result preview is a list so JSON decoding was failing

Reply:
It might be that the `try`/`except` handles the case of a failed run. The Run API doesn't offer a very clear contract in this case at the moment.

Reply:
Yeah I'm not sure to be honest. Looking at the [type annotation](https://github.com/mystic-ai/pipeline/blob/main/pipeline/schemas/run.py#L92) for the `result_preview` it is supposed to be a `list` or `dict` and `json.loads` expects a `str` so this would never work. Perhaps we were trying to serialise the `dict` into a string instead? In any case, I will ignore the JSON stuff and add a `try/except` to catch the case where there is no `result_preview`